lib/model/field.dart

```diff
@@ -32,6 +33,7 @@ class Field {
     this.fromJson,
     this.toJson,
     this.defaultValue,
+    this.disallowNull = false,
```

Comment:
shouldn't this be true? Because https://github.com/icapps/flutter-model-generator/pull/112/files#diff-ac9739cbe4a87fb64e4f084465adfa916d3d991a4e00c54b458e90a4a8fd9864L103 is default true

Reply:
That's the semi breaking change to flip if from the default